etna/analysis/feature_selection/mrmr_selection.py

```diff
@@ -59,6 +71,10 @@ def mrmr(
     fast_redundancy:
         * True: compute redundancy only inside the the segments, time complexity :math:`O(top\_k * n\_segments * n\_features * history\_len)`
         * False: compute redundancy for all the pairs of segments, time complexity :math:`O(top\_k * n\_segments^2 * n\_features * history\_len)`
+    drop_zero:
+        If True, drop features with zero relevance before MRMR. If top_k is greater number of features
```

Comment:
It is not clear for me. If top_k is greater than number of features with relevance >0, select all non-zero relevance features with randomly selected zero relevance features.

Reply:
I select all non-zero relevance features and add some features with zero relevance in order to select exact top_k features

Reply:
I mean what is written is very unclear and complicated. Could you rewrite more clear?